airbyte-integrations/connectors/source-zenloop/integration_tests/catalog.json

```diff
@@ -25,6 +25,26 @@
       "source_defined_cursor": true,
       "default_cursor_field": "test",
       "json_schema": {}
```

Comment:
Since you're entirely refactoring integration tests, fill in the `json_schema` properties for all streams in `catalog.json` to be completely consistent.

Reply:
added